tox.ini

```diff
@@ -15,11 +15,10 @@ description =
 deps =
   ansible28: ansible>=2.8,<2.9
   ansible29: ansible>=2.9,<2.10
-  ansibledevel: ansible-base @ git+https://github.com/ansible/ansible.git@devel#egg=ansible-base==2.10.0
-  # ATTENTION: The following tarball is a temporary ACD incarnation
-  # ATTENTION: and may be removed/disappear at any time.
-  # FIXME: Replace this once ACD is out:
-  ansibledevel: ansible @ https://toshio.fedorapeople.org/ansible/acd/ansible/ansible-2.10.0.tar.gz#egg=ansible==2.10.0
```

Comment:
Mixing Toshio's pre-packaging with pypi was not the best idea. That should be more reliable until we will switch to pure pip.